code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -487,7 +487,7 @@ var/global/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","E
 /mob/proc/changeling_unstun()
 	set category = "Changeling"
 	set name = "Epinephrine Sacs (45)"
-	set desc = "Removes all stuns"
+	set desc = "Removes all stuns. Don't use it in rapid succession."
```

Comment:
Careful, descs can get truncated in the UI.

Reply:
Checked, it's not too long.